pytorch_lightning/loops/fit_loop.py

```diff
@@ -54,6 +55,7 @@ def __init__(
         self.max_epochs = 1000 if (max_epochs is None and max_steps is None) else max_epochs
         self.min_epochs = 1 if (min_epochs is None and min_steps is None) else min_epochs
         self.training_loop = TrainingEpochLoop(min_steps, max_steps)
+        self.validation_loop = EvaluationDataLoaderLoop()
```

Comment:
@awaelchli has it been decided that we do the full name instead of the shorter one?

```suggestion
        self.val_loop = EvaluationDataLoaderLoop()
```

Reply:
no but highest priority is consistency over anything else imo

Reply:
Is that an argument in favor of the long or short name?